docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,95 @@
+![immutadot logo](https://raw.githubusercontent.com/Zenika/immutadot/master/misc/otter.svg?sanitize=true)
+=================
+
+# Getting started
+
+As you already know immutad●t is a javascript library that help you dealing with nested immutable structure. The easiest way to try immutad●t is to use [runkit](https://npm.runkit.com/immutadot). You can test all the following examples in it. If you rather use immutad●t in local, checkout out the [Installation](https://raw.githubusercontent.com/Zenika/immutadot/master/README.md) section of our README.
+
+## Setting nested properties
+
+ES2015+ provides you all necessary tools to keep nested structures immutable. The [spread operator](https://mdn.io/Spread_operator) is a succinct syntax create new arrays and objects using existing ones.
```

Comment:
What about :
[ES2015+ destructuring](https://mdn.io/Spread_operator) provides you all necessary tools to keep nested structures immutable: